[Content] The Avgi - Atomic Space Dragonflies, In Space (#9172)
endless-sky/endless-sky: data/avgi/avgi 1 first contact.txt

			`	"Reservation. Perhaps Captain <last> will be able to find a solution." says Galano. "I would not dismiss the possibility too quickly."`
			choice
				`	"While I'm here, can I buy your technology?"`
					goto buystuff
				`	"Is there anything I can do to help in the meantime?"`

			`	"Consideration. As an alien, you will inevitably be a novelty to most of us, but the existence of humans is well-known among the public enough for you to walk among us without issue. I will speak to others about a more formal role, but until then I am unsure if we can ask anything of you beyond keeping the peace."`
			label buystuff
			`	Galano says, "Interest. I would expect human technology to be superior to our own in most aspects, but we will of course allow you to purchase items with your human credits, which we will back with our own currency with the agreement of the Assembly. In the event that it becomes possible to open trade relations with your human government, we may find it useful to have a small stockpile of your currency available."`
			`	"Amusement. Not to mention the goodwill such a gesture could buy us," says Asime.`

> 			`	Prasinos says, more seriously, "Reservation. Our material resources are quite limited at the moment, so for now I would only support providing for the requisition of small items that do not strain the needs of the Guard. No military goods, at least not now."`

Reviewer: They comment here that they would not provde military goods, yet several of their weapons are immediately available for purchase. I feel like there should either be a clarification as to which weapons are meant by this or all the weapons should be put behind a license.

			label end
			`	You sit together in silence for a few more minutes, when Darius seems to receive a message on his communicator. "I get patched in to the emergency alert system," he explains as he reads it. Suddenly, he pales. "A scout is reporting a massive invasion headed towards the Ensemble system. Dozens of Aberrations, flying straight through the Shroud." He looks up at you. "They don't normally try to go too deep in the Shroud. Something about it spooks them, I think, but Ensemble is shallow enough that they've attacked it before. Nothing of this scale though, and... damn, Sora's on Weledos, the moon in the Ensemble system."`
			`	He looks at you, hopeful. "We've got to do something. The Avgi are assembling a relief force around the Navigeo Yards, their big shipyard not too far from here. Come on, let's join it."`
				accept

> mission "Avgi: Defend Ensemble 2"

Reviewer: I'd also have a larger look at this mission. You have to travel quite a ways to the battle, and it is very likely that the common random spawns or unfortunate positioning when jumping in will lead to you, or the rest of the fleet following you, getting nuked.
Personally, I had to do around 8 runs just to make it to the system with most of the fleet intact, and 13 to actually survive the battle. (This was using a Bactrian with 3 olofez'.) I may have just been unlucky many of those times, but the setup of the mission (having a bunch of nuke happy ships travel a long distance with you) leads to a high likelyhood of something going wrong in a manner that leads to your or their deaths.

I think reducing the travel distance would be the single most impactful change to make this mission less tedious to play, other than that I might look at the fleet composition (the glacial turn speed of the "Atenuatia (Atomic)" variant contributed the most to this issue) or possibly reduce the range of the missiles slightly.

Reviewer 2: Maybe switch the Atomic variant's powerful thrusters for powerful steering?

Reviewer: seems like the steering was switched out for the nukes. I don't think downgrading the engine would be particularly good because then it would have either power or thrust problems (depending on how you outfit it.) Sacrificing the cargo would probably get you there.

Reviewer: I just realized some things
1. that wouldn't work because the Ophrys stockpiles take up basically all the cargo space.
2. All it needs to have enough space for the larger steering is one less Ophrys stockpile, it has a little less endurance with a full compliment fo nuclear fighters, but is still very functional. (also by the time the stocks run out in testing I am usually only left with one or two fighters/drones operational (because the rest nuked themselves.)

			choice
				`	"I have a jump drive that lets me travel beyond the hyperlanes, but it won't let me jump back to human space."`
				`	"I have a device that lets me go where I please, but it isn't working at the moment."`

			`	"Regret. Then the situation is most unfortunate," she says. "Commitment. We will offer you what help we can, but I did not wish to lie to you about their predicament, or provide you with false hope."`
			`	"Reservation. Perhaps Captain <last> will be able to find a solution." says Galano. "I would not dismiss the possibility too quickly."`
			choice
				`	"While I'm here, can I buy your technology?"`
					goto buystuff
				`	"Is there anything I can do to help in the meantime?"`

			`	"Consideration. As an alien, you will inevitably be a novelty to most of us, but the existence of humans is well-known among the public enough for you to walk among us without issue. I will speak to others about a more formal role, but until then I am unsure if we can ask anything of you beyond keeping the peace."`
			label buystuff

> 			`	Galano says, "Interest. I would expect human technology to be superior to our own in most aspects, but we will of course allow you to purchase items with your human credits, which we will back with our own currency with the agreement of the Assembly. In the event that it becomes possible to open trade relations with your human government, we may find it useful to have a small stockpile of your currency available."`

Reviewer 3: ```suggestion
			`	Galano says, "Interest. I would expect human technology to be superior to our own in most aspects, but we will of course allow you to purchase our items with the agreement of the Assembly. In the event that it becomes possible to open trade relations with your human government, we may find it useful to have prior transactions."`
```
The problem with this is that it creates a rabbit hole and addresses the elephant in the room regarding credits. We want to keep them a simple gameplay mechanic and keep it universal, so it's better not to talk about any direct currency conversions, as then it would be the question for all non-human factions. This suggesting makes it more generalized and still retains the idea that there is some "investment" to be had, without specifying anything.

# Copyright (c) 2023 by Azure
#
# Endless Sky is free software: you can redistribute it and/or modify it under the
# terms of the GNU General Public License as published by the Free Software
# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
# PARTICULAR PURPOSE. See the GNU General Public License for more details.
#
# You should have received a copy of the GNU General Public License along with
# this program. If not, see <https://www.gnu.org/licenses/>.

> mission "Avgi: Lost in Twilight"

Reviewer 4: You need to disable this in the integration tests. (Along with any other always active missions you are adding.)

	invisible
	landing
	to complete
		never
	on enter
		system
			attributes "twilight" "tangled shroud"
		log "Became trapped in a strange region after entering a system my jump drive could not return from."
		fail
		set "avgi: lost in twilight"
		dialog phrase "lost in twilight"

phrase "lost in twilight"
	word

> 		`As soon as you jump into the system, you notice something is wrong. The characteristic plasma of your jump drive fades unnaturally quickly, and as you check your ship's systems, you notice its temperature is somehow much lower than normal. You attempt to plot a jump back to where you came from, but your computer tells you the system is out of range. It appears you are stuck in this region for now.`

Reviewer 4: Who says I do any of this?
```suggestion
		`As soon as you jump into the system, you notice something is wrong. The characteristic plasma of your jump drive fades unnaturally quickly, and as you check your ship's systems, you notice its temperature is somehow much lower than normal. Your computer indicates that the system you came from is out of range. It appears you are stuck in this region for now.`
```

mission "Avgi: First Contact"
	autosave
	name "An Alien Encounter"
	description "Accompany the alien ship to the destination marked on your map."
	landing
	source
		attributes "avgi"
	destination "Peripheria"
	mark "Arche"
	mark "Ballad"
	clearance
	on offer
		set "avgi: aberrant siege"
		event "twilight label"
		event "tangled shroud label"
		conversation
			branch aviskir
				has "flagship planet: Aviskir"
			branch coreworld
				has "flagship planet attribute: avgi core"

> 				not "flagship planet attribute: station"

Reviewer 4: There are only two planets with "avgi core", Aviskir and Livolua, and neither of them have "station".
You're already branching away for Aviskir, so this branch is basically just `has "flagship planet: Livolua"`. Are you expecting the results of these filters at the time of this mission being offered to be different later? If not, it would be much clearer to readers to just use Livolua here.

			label uninhabited
			`While this world looks presently unsettled, you can clearly make out signs that it has been explored, and even exploited, by an alien civilization. However, the signs also show that something disastrous happened here. Ruined satellites litter low orbit, and you can detect the broken remnants of surface outposts smeared across the land below. You take the <ship> in for a close pass over one of the structures, and can see small craters and patches of glassed terrain scattered around it.`
				goto orbit

			label orbit
			`	Back in orbit, you are startled when another ship appears over the horizon on an intercept trajectory with you. It doesn't look like any other ship you've seen before, appearing more than a little ungainly compared to the sleeker human ships you're accustomed to. You aren't sure whether it belongs to the same species that inhabits this space, or an entirely different one, but it at least definitely isn't similar to the hostile ships that seem to infest this star cluster.`
			choice
				`	(Try transmitting a standard hail.)`
					goto talk
				`	(Transmit a series of prime numbers.)`
					goto prime
				`	(Remain silent.)`
					goto silent

> 			label station

Reviewer 4: Is the following text supposed to be used for all stations regardless of if they are inhabited?

		log "Factions" "Avgi" `The Avgi are a young race of aliens resembling large, colorful dragonflies, which inhabit the Twilight Nebula in this part of the galaxy. After an overwhelming invasion by ships they call the Aberrant, most of their surviving spacecraft fled south through a maze-like region called the Tangled Shroud.`
		log "Factions" "Avgi Consonance" `The exiled Avgi seem to be led by a central government called the Consonance, but there are many who do not recognize this government.`
		log "People" "Asime" `Asime is one of the three First Speakers of the Consonance Assembly. While reserved, Asime appears to be defensive about the decisions made by the Consonance in response to the Aberrant incursion.`
		log "People" "Galano" `Galano is one of the three First Speakers of the Consonance Assembly. He appears to be an elder statesman, with much pride for what the Avgi have accomplished.`
		log "People" "Prasinos" `Prasinos is one of the three First Speakers of the Consonance Assembly. She seems to take a negative view on the Avgi history of division and fractiousness.`
		event "outer limits label"

> 		conversation

Reviewer 4: This conversation (and associated log entries" ought to go in the "on offer" of the next mission.

			choice
				`	"Hello there."`
				`	"Greetings, human. I come in peace."`
			`	The man blinks in surprise, then extends a hand, which you shake. "So, another one of us here in Fairyland. I don't know if I should be glad to see another friendly face, or feel bad that another soul is stranded here along with us." He shakes his head. "Let's find somewhere to talk. If you don't mind telling me, how'd you end up here anyway?"`
			choice
				`	"I have a jump drive."`
				`	"I have a device that can bypass hyperlanes."`
			`	He somehow manages to look even more astonished as you walk. "A jump drive? Where the hell did you get one of those?"`
			choice
				`	"I took it from some hostile aliens near the Core."`
					goto korath
					to display
						"reputation: Korath" < -10
				`	"I took it from some aliens in the Deneb system."`
					goto pug
					to display
						has "FWC Pug 2D: done"
				`	"I got it from the Syndicate, who took it from an alien race near the Core."`
					goto korath
					to display

> 						has "FW Pug 2B: done"

Reviewer 4: If you already have one, the Syndicate won't give you one.
```suggestion
						has "FW Pug 2B: done"
						not "FW Pug 2C: Jump Drive: done"
```

				`	"I took it from some angry space squirrels."`
					goto unfettered
					to display

> 						has "Wanderers: Translation Machine: done"

Reviewer 4: You can get a jump drive from the Unfettered without having done this mission, or even having visited Wanderer space,

			coward secretive staying target unconstrained uninterested vindictive
		system "Empyrean"
		fleet
			names "aberrant names"
			variant
				"Aberrant Hugger"
				"Aberrant Pileup"
	npc kill
		government "Aberrant"
		personality
			confusion 40
			coward secretive staying target unconstrained uninterested vindictive
		system "Firmament"
		fleet
			names "aberrant names"
			variant
				"Aberrant Triplet"
	npc kill
		government "Aberrant"
		personality
			confusion 40
			coward secretive staying target unconstrained uninterested vindictive
		system "Lumine"
		fleet
			names "aberrant names"
			variant
				"Aberrant Chomper" 2

	on complete
		conversation
			`You receive a hero's welcome back on the Feo Platform from the Twilight Guards stationed there. After a short celebration and many expressions of praise toward you, an alien stranger, you find Darius and Sora waiting by the <ship> to speak with you.`
			`	"<first>, I think we have a golden opportunity," Darius says. "I was just talking to an Assemblyman, Hriso, who's worried about the political situation in the eastern branch of the Tangled Shroud."`
			`	"A bunch of selfish cowards, they are," says Sora disdainfully.`

> 			`	"Something about the miners grumbling about price controls on nickel-iron," Darius says after a moment, "but he wants to know if you could use your jump drive to take him there for some outreach to the miners without needing to spend weeks wandering through the mess of twisted hyperlanes. It's the perfect chance to meet my friend and ask him some questions. We'll need to head to Peripheria to pick up Hriso and get this trip sorted out."`

Reviewer 4: This should also be in the "on offer" of the next mission, especially because that mission requires passenger space and this does not, so a person might see this conversation but not have a new mission.

			attributes "twilight" "tangled shroud"
		fail
		set "avgi: twilight trapped"
		dialog phrase "twilight trapped"

phrase "twilight trapped"
	word
		`As soon as you jump into the system, you notice something is wrong. The characteristic plasma of your jump drive fades unnaturally quickly, and as you check your ship's systems, you notice its temperature is somehow much lower than normal. You attempt to plot a jump back to human space, but your computer tells you the system you just came from is out of range. It appears you are stuck in this region for now.`

mission "Avgi: First Contact"
	autosave
	name "An Alien Encounter"
	description "Follow the alien ship to the destination marked on your map."
	source
		attributes "avgi"
	landing
	waypoint "Arche"
	waypoint "Ballad"
	destination "Peripheria"
	clearance
	on offer
		event "tangled shroud label"
		conversation
			branch aviskir
				has "flagship planet: Aviskir"
			branch coreworld
				has "flagship planet attribute: avgi core"
				not "flagship planet attribute: station"
			branch station
				has "flagship planet attribute: station"
			branch uninhabited
				has "flagship planet attribute: uninhabited"

			label aviskir

> 			`The first thing you notice about this earth-like world is the utter devastation in orbit. Your ship's sensors pick up the remains of what appear to be dozens of wrecked stations and countless destroyed satellites, forming a large swath of debris scattered in various orbits. A few larger objects are mostly intact, but lifeless, with no signal emissions or heat signatures. As your ship touches the upper atmosphere, you can see, even with your naked eye, glittering fragments of an immensely long tether slowly spinning in orbit.`

Reviewer 2: Based on [this NASA page](https://science.nasa.gov/universe/exoplanets/earth-like-planets-have-earth-like-interiors/), I would assume it should be capital. However, you probably should leave it open and let other people see it before committing it.

				goto orbit

			label scratched
			`	You make it no more than a few seconds before the <ship> is lanced by a rapid succession of shots from below. Your ship easily shrugs off the kinetic impacts, but the species that lives here doesn't seem willing to welcome you. When you detect the launch of a rather unreasonable number of missiles, you decide it would be safer to proceed back to orbit.`
				goto orbit

			label uninhabited
			`While this world looks presently unsettled, you can clearly make out signs that it has been explored, and even exploited, by an alien civilization. However, the signs also show that something disastrous happened here. Ruined satellites litter low orbit, and you can detect the broken remnants of surface outposts smeared across the land below. You take the <ship> in for a close pass over one of the structures, and can see small craters and patches of glassed terrain scattered around it.`
				goto orbit

			label orbit

> 			`	Back in orbit, you are startled when another ship appears over the horizon, on an intercept trajectory with you. It doesn't look like any other ship you've seen before, appearing more than a little ungainly compared to the sleeker human ships you're accustomed to. You aren't sure whether it belongs to the same species that inhabits this space, or an entirely different one, but it at least definitely isn't similar to the hostile ships that seem to infest this star cluster.`

Reviewer 2: True.

> 			choice

Reviewer 2: Good point.

			`	Following the instructions of the station's control authority, you fly through the station's open frame to its small pressurized area, which houses the construction workers and maintenance crews. You've been told that word of your arrival has been sent ahead of you, so you aren't entirely surprised to see a middle-aged human waiting in the hangar, a look of pure astonishment on his face as your ship settles into a free hangar. You disembark from the <ship> to find Darius waiting right outside the hatch.`
			choice
				`	"Hello."`
				`	"Hey there."`
				`	"I come in peace."`
			`	The man blinks in surprise, then extends a hand, which you shake. "So, another one of us here in Fairyland. I don't know if I should be glad to see another friendly face, or feel bad that another soul is stranded here along with us." He shakes his head. "Let's find somewhere to talk. If you don't mind telling me, how'd you end up here anyway?"`
			choice

> 				`	"I have a jump drive."`

Reviewer 2: Kind of the point, because it goes to the same label, but your choice whether to put it in or not. Just being able to say one thing is my great though.

			choice
				`	"How'd you end up in Avgi space without a Jump Drive?"`
			label story
			`	"Funny story, that," he says. "We were part of a big convoy from Valhalla, sponsored by one of the major universities there. I was working with Deep Security, after my stint in the Navy... ended." He trails off. "Anyway, we were in a Mule carrying some scanning equipment. To research the Quarg, you see. Officially we were there for cultural outreach and exchange, you know, the usual, but really we wanted some good scans of a Wardragon. The Quarg don't really approve of it, but they don't do much to discourage it either."`
			`	"So, we were on our way back, just entering Dabih. Really successful trip and everything, with a big bonus waiting for me thanks to how long the trip across the entire Republic took. But then our scanners started picking up weird readings from particular hyperlanes, and we started hearing some pirates chattering about how something weird was happening in the Sadr system. A new hyperlane, pointing south, they said. It was just a few jumps away from our current location, and we were the only Deep ship in the area that could go off alone."`

> 			`	He looks at his hands. "So I... I made the decision, as captain. I volunteered my crew to investigate it, to travel through that hyperlane in the name of science. For what the Deep holds highest, because I guess I wanted... I don't know. Not the prestige. But to live up to what the Deep's supposed to be, the ones at the forefront of human exploration and research. They took me in when I had nowhere else to go, and I guess I wanted to repay that somehow. And now, now a dozen good men and women are dead, and the rest are stuck here behind a hyperlane that closed the moment we went through it, living among aliens fighting a war of extinction that they probably can't win."`

Reviewer 2: ```suggestion
			`	He looks at his hands. "So I... I made the decision, as captain. I volunteered my crew to investigate it, to travel through that hyperlane in the name of science. For what the Deep holds highest, because I guess I wanted... I don't know. Not the prestige. But to live up to what the Deep is supposed to be, the ones at the forefront of human exploration and research. They took me in when I had nowhere else to go, and I guess I wanted to repay that somehow. And now, now, a dozen good men and women are dead, and the rest are stuck here behind a hyperlane that closed the moment we went through it, living among aliens fighting a war of extinction that they probably can't win."`
```

			`	You ask, "The other members of your crew, where are they now?"`
			`	"There are a few of us left," he sighs, "but, well, I'm not on the best of terms with them. They blame me for bringing them here, and I can't fault them for it."`
			choice
				`	"That seems unfair. There's no way you could have known what would happen."`
					goto blame
				`	"Is there a chance of reconciling with them?"`

> 			`	"I've made... a lot of mistakes. Ones with a price paid in lives. But," he looks down, "...maybe. They don't hate me or anything like that, we just... keep our distance. Sora's off doing who knows what with the Twilight Guard on Weledos, and Leon is on and off with those mining guilds to the East. There are a few others left too, but like I said. A price paid in lives."`

Reviewer 2: ```suggestion
			`	"I've made... a lot of mistakes. Ones with a price paid in lives. But," he looks down, "... maybe. They don't hate me or anything like that, we just... keep our distance. Sora's off doing who knows what with the Twilight Guard on Weledos, and Leon is on and off with those mining guilds to the East. There are a few others left too, but like I said. A price paid in lives."`
```

Reviewer 2: Everywhere else it is that way...

			`	"Caution. Atomics will be used in the upcoming battle, so stay your distance from our warheads," she says.`
			choice
				`	"Acknowledged."`
				`	"Wait, nuclear weapons? You people use those?"`
			`	She turns away from the viewscreen as another Avgi starts speaking to her, dismissing you.`
				accept
	npc
		government "Avgi (Twilight Guard)"
		personality heroic
		fleet
			names "avgi military names"
			variant
				"Interferos"
				"Prismaios"
				"Prismaios (Atomic)"
				"Tremoros" 3
	npc
		government "Avgi (Twilight Guard)"
		personality heroic waiting
		fleet
			names "avgi military names"
			variant
				"Difraktos (Atomic)"
				"Difraktos (Missile)"
				"Refraktos"
				"Atenuatia (Atomic)"
				"Vibratia" 22
				"Photikon (Atomic)" 6
	npc
		government "Avgi (Twilight Guard)"
		personality heroic staying waiting
		system "Ensemble"
		fleet
			names "avgi military names"
			variant
				"Prismaios"
				"Tremoros" 6
				"Difraktos (Kinetic)"
				"Refraktos"
				"Diaspersi"
				"Vibratia" 8
				"Photikon" 2
	npc evade
		government "Aberrant"
		personality
			confusion 60

> 			hunting coward vindictive staying unconstrained secretive

Reviewer 2: ```suggestion
			coward hunting secretive staying unconstrained vindictive
```

			`	"Explanation. We send scouting teams and patrols into the space controlled by the Aberrant regularly. Every expedition beyond Evanescence is essentially behind enemy lines, cut off from resupply and in many cases communication. As you might imagine, such adventures are fraught with danger, and it is unfortunately not unheard of for a scout ship to be lost."`
			choice
				`	"Yes, I've found that exploration is often dangerous."`
					goto exploration
				`	"Sounds like you should build stronger scout ships."`
			`	"Agreement. The Polari are a fine line of vessels, but not one made for the environment of war. We have experimented with using the Prismaios as an intelligence platform in more contested environments, but it lacks the versatility and drone bays that has made the Polari so venerable in the first place."`
				goto point
			label exploration
			`	He flicks his wings. "Regret. It is a sad truth, but those on the forefront of expanding our knowledge are often those who face the most danger from the unknown."`
			label point

> 			`	"But, to the point. A scout team, aboard the Polari-class scout 'Silver Albedo', was sent to survey the Twirl system two weeks ago, but has missed its check in by multiple days. To reach that star cluster and return would normally require a trip of weeks, all thorough perilous territory, and I am ashamed to say that we could be forced to consider writing off the scouts as lost. But I have heard that the visiting human, Captain <last>, is in possession of a new kind of faster-than-light drive, one that can travel without the need for starways. Tell me, is this true, question?"`

Reviewer 2: ```suggestion
			`	"But, to the point. A scout team, aboard the Polari-class scout 'Silver Albedo,' was sent to survey the Twirl system two weeks ago, but has missed its check-in by multiple days. To reach that star cluster and return would normally require a trip of weeks, all through perilous territory, and I am ashamed to say that we could be forced to consider writing off the scouts as lost. But I have heard that the visiting human, Captain <last>, is in possession of a new kind of faster-than-light drive, one that can travel without the need for starways. Tell me, is this true, question?"`
```

			`	You nod.`

> 			`	"Excitement. Then, the trip could be shortened to mere days, giving us the possibility of mounting a rescue. I will mark the location of the Twirl system on your map, though it is probable that the Silver Albedo is elsewhere in the star cluster." The highest priority is of course the safe recovery of its crew, though if you could recover any data or samples they have managed to gather that would also be much appreciated.`

Reviewer 2: ```suggestion
			`	"Excitement. Then, the trip could be shortened to mere days, giving us the possibility of mounting a rescue. I will mark the location of the Twirl system on your map, though it is probable that the Silver Albedo is elsewhere in the star cluster. The highest priority is of course the safe recovery of its crew, though if you could recover any data or samples they have managed to gather, that would also be much appreciated."`
```

				accept

	npc assist

> 		government "Derelict"

Reviewer 2: ```suggestion
		government "Avgi"
```

> 		personality derelict target pacifist mute

Reviewer 2: ```suggestion
		personality derelict mute pacifist target
```

		system Whirlwind
		ship "Polari (Derelict)" "Silver Albedo"
		conversation

> 			`You maneuver alongside with the Silver Albedo, which is covered in scorch marks and other signs of damage. After transmitting a code given to you by Captain Kokkino, you wait for any signs of life. After a few minutes you receive a response, which you pass through your translator.`

Reviewer 2: ```suggestion
			`You maneuver alongside the Silver Albedo, which is covered in scorch marks and other signs of damage. After transmitting a code given to you by Captain Kokkino, you wait for any signs of life. After a few minutes you receive a response, which you pass through your translator.`
```

			choice
				`	"Where do I come in?"`
				`	"What do you want me to do?"`
			`	"Consideration. Your ship is far beyond anything we can produce indigenously, at least for the moment. While you are certainly skilled in fleet combat, I am of the opinion that you would be best put to use launching independent strikes, those that a single one of our vessels could not manage on their own."`
			`	He looks down at a device similar to a tablet. "With the attack on Ensemble, we were forced to pull back our main fleetgroups from the front lines in Quintessence in an attempt to contain them. While we have since reclaimed our initial positions, during the maneuvering a large group of Aberrant were able to slip past us, not into the Shroud, but rather north, towards the Empyrean system. With the current situation, I do not want to divert an entire strike force from the defenses to hunt them down. Report back to the Feo Platform when the Aberrant are eliminated."`
				accept

	npc kill
		government "Aberrant"
		personality
			confusion 90

> 			coward vindictive staying unconstrained uninterested secretive target

Reviewer 2: ```suggestion
			coward secretive staying target unconstrained uninterested vindictive
```

		system "Corporeal"
		fleet
			names "aberrant names"
			variant
				"Aberrant Junior"
				"Aberrant Traincar"
	npc kill
		government "Aberrant"
		personality
			confusion 90

> 			coward vindictive staying unconstrained uninterested secretive target

Reviewer 2: ```suggestion
			coward secretive staying target unconstrained uninterested vindictive
```

		system "Empyrean"
		fleet
			names "aberrant names"
			variant
				"Aberrant Hugger"
				"Aberrant Pileup"
	npc kill
		government "Aberrant"
		personality
			confusion 90

> 			coward vindictive staying unconstrained uninterested secretive target

Reviewer 2: ```suggestion
			coward secretive staying target unconstrained uninterested vindictive
```

		system "Firmament"
		fleet
			names "aberrant names"
			variant
				"Aberrant Longfellow" 2
				"Aberrant Dancer"
	npc kill
		government "Aberrant"
		personality
			confusion 90

> 			coward vindictive staying unconstrained uninterested secretive target

Reviewer 2: ```suggestion
			coward secretive staying target unconstrained uninterested vindictive
```

	source "Outpost Tekis"
	passengers 3
	blocked "You'll need three free bunks on the <ship> to transport Darius, Sora, and Leon."
	destination "Valhalla"
	to offer
		has "Avgi: Dissonance Outreach: done"
	on enter "Decet"
		conversation
			`	Just as your jump drive reacted strangely when you first entered Avgi space, it does so again, but with an inverse effect. Your instrumentation say it is much warmer than usual, almost hot enough to be dangerous. Perhaps your jump drive's range has changed as well.`
	on enter
		system
			attributes "core" "deep" "dirt belt" "near earth" "rim" "south"
		conversation
			`	As the purple flames of your jump drive fade, Darius, Sora, and Leon are all speechless. For the first time in six years, they are back in human space.`
			`	"<first>," says Darius in a shaky voice. "Take us to Valhalla, please. I need to see my home."`
	on complete
		clear "avgi: twilight trapped"
		"reputation: Avgi (Dissonance Aggressive)" <?= -10
		conversation
			`	You land in Valhalla's main spaceport, and Darius practically jumps out of your ship before it can even touch down. "I'll be in contact, <first>, but I have to tell someone that I'm still alive."`

> 			`	Now that you know of a way both in and out of the Twilight, you can visit the Avgi without fear of being trapped again.`

Reviewer 2: ```suggestion
			`	Now that you know of a way both in and out of the Twilight, you can visit the Avgi without fear of being trapped again.`

```

			`	Asime adds, "Truth. And we are what the Avgi need, now that all of that could be lost to the Incursion, to the Aberrant it spawned. We need what humanity has, a united Republic to lead all of our people."`
			choice
				`	"How do you know of humanity?"`
					goto humans
					to display
						not "avgi: asked about humans"
				`	"The Aberrant?"`
					goto aberrant
					to display
						not "avgi: asked about aberrant"
				`	"That's all the questions I have. What happens now?"`
					goto conclusion

			label humans
			action
				set "avgi: asked about humans"

> 			`	"Hesitance. It may come as a surprise to you, but you are not the first human to come to our space. Three- no, that would be six of your years ago, an alien ship was found drifting around a star in our space, swarmed by Aberrations. By some miracle they were driven away, and the ship recovered. And those were the first, and until now, the only humans we have had the honor of meeting. A crew of explorers from, I believe they said, "the depths," though I am not quite sure if that is a planet or a region of space."`

Reviewer 2: Sorry, this had been correct.
```suggestion
			`	"Hesitance. It may come as a surprise to you, but you are not the first human to come to our space. Three- no, that would be six of your years ago, an alien ship was found drifting around a star in our space, swarmed by Aberrations. By some miracle they were driven away, and the ship recovered. And those were the first, and until now, the only humans we have had the honor of meeting. A crew of explorers from, I believe they said, 'the depths,' though I am not quite sure if that is a planet or a region of space."`
```

# Copyright (c) 2023 by Azure
#
# Endless Sky is free software: you can redistribute it and/or modify it under the
# terms of the GNU General Public License as published by the Free Software
# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
# PARTICULAR PURPOSE. See the GNU General Public License for more details.
#
# You should have received a copy of the GNU General Public License along with
# this program. If not, see <https://www.gnu.org/licenses/>.

mission "Avgi: Trapped in Twilight"
	invisible
	landing
	to complete
		never
	on enter
		system
			attributes "twilight" "tangled shroud"
		fail
		set "avgi: twilight trapped"
		dialog phrase "twilight trapped"

phrase "twilight trapped"
	word

> 		`As soon as you jump into the system, you notice something is wrong. The characteristic plasma of your jump drive fades unnaturally quickly, and as you check your ship's systems, you notice its temperature is somehow much lower than normal. You attempt to plot a jump back to human space, but your computer tells you the system you just came from is out of range. It appears you are stuck in this region for now.`

Reviewer 5: Actually, there might be a problem. If you enter Avgi space in the right place- Peak system, accessible from Sadr- and you have Decet visible as unexplored on your map (as you will if you've visited Sadalsuud system, maybe more), and you try to plot a course back to human space... it will show you a path via Flutter to Decet and right out of the Twilight. I'm not sure how one would move Decet or Flutter to make this not happen, unfortunately...

Additionally, if one goes directly to Decet this fires with the path out being completely known.

Reviewer 5: Ah, sorry- it goes through Octet too. Apologies for not being clearer.

> mission "Avgi: First Contact"

Reviewer 5: I assume this is intentional- Distant Shores vouching for you, perhaps- but you can totally land on Aviskir and get refueled/repaired. It's a little funny after having read the mission here to not have to worry about battery fire....

			`	Eventually, after a number of turns and descents, you pass through an open archway into some kind of conference room. Large and circular, with a high ceiling, it is occupied by a trio of multicolored dragonfly aliens, each perched on a metal ring running around a circular table.`
			choice
				`	"Hello."`
				`	"Who are you?"`
			`	The alien directly across from you, gray-white in color, begins speaking with the aid of another of the translator devices, this time taking on a feminine voice.`
			`	"Invitation. I apologize for the inconvenience of traveling so far from your ship to meet us, but I was advised against being present for your arrival for fear of any unexpected hostilities, as small as such chances were estimated. My name, for lack of a direct translation, is 'Asime,' and with me are Speakers Prasinos and Galano. Together, we represent the Avgi Consonance as the First Speakers of the Assembly of Polities."`

> 			`	Prasinos, in grass-green, and Galano, a milky-white Avgi that looks vaguely aged somehow, both wave at you with their forelimbs in a surreal imitation of the human gesture.`

Reviewer 5: Their name means "green", no their body is probably green.

			`	"Amazing. So, your jump drive can get us out of here then?"`
			choice
				`	"I'm afraid not. It can't return to human space for some reason."`
				`	"No, but I'm not giving up on finding a different way out."`
				`	"Nope!"`
			`	"Damn. That would be too easy, I guess. But it's been too long since I've seen a fresh human face. I've been working for the Twilight Guard here for months, trying to build up a second line of defenses in case that station in Arche falls. At least today's battle shows those defenses, together with a supporting fleet, can hold. You mind if I tag along with you, in case you do manage to find a way out?"`
			choice
				`	"Of course."`
					goto yes
				`	"Sure, as long as Darius doesn't mind."`
			`	You look over at Darius, who responds after a few seconds of silence. "Of course you can come along, Sora."`
			label yes
			`	"Sounds good to me, then. I'll see you around when you're ready to leave." She walks off to collect her belongings.`
			`	A few hours later, an Avgi cruiser, a Refraktos-class Laserstar, settles down on the pad next to yours, and an Avgi officer disembarks.`

> 			`	"Salutations. Hello, Captain <first> <last>. My name is Kokkino, Captain of the Distant Collision. Tell me, is the <ship> a warship, question?"`

Reviewer 5: Probably not worth it change things to specify the Distant Collision is around as part of the fleet during the battle just for the consistency, but the thought came to mind.

			choice
				`	"Yes, of course."`
				`	"I'll hear you out."`
				`	"I'd rather not."`
					decline
			`	"Enthusiasm. Your help will save many lives. Then, if you would, meet me in the spaceport in an hour." He gives you a set of directions to a small building in Ensemble's main spaceport before leaving. "We will have much to discuss."`

			label end
			`	Darius exits the <ship> behind you. "What was that about?" he asks.`
			choice
				`	"It sounds like they'd like my help with something."`
				`	"I'm going to help them with the war effort."`
			`	He frowns. "I see. When do you think we'll be able to head up north to look for a way out?"`
			choice
				`	"We'll do it soon, I promise."`
				`	"I'm not sure."`
			`	He looks a little disappointed, then collects himself. "Well, I suppose there's no real urgency to it. Though, I've been thinking. If your jump drive wasn't able to let you escape from the north, there might not be a point in looking in that direction."`
			`	"What do you suggest?" you ask.`

> 			`	"I have a friend among the Avgi, one who doesn't frequent the official circles in the Consonance as much. There's an entire second branch of the Tangled Shroud, mostly ran by miners and small-time shipbuilders, the kind of people who talk a lot among themselves about rumors and tall tales. The official story behind the Incursion, the invasion of the Aberrant, is that they came from a chain of systems in the East, but no one ever talks about where exactly that chain leads. I think we should pay my friend a visit after you wrap up this next mission."`

Reviewer 5: At the moment, the Shroud doesn't seem to branch quite as strongly as this implies? The most direct path through it seems to go through where I'd expect this second branch to be.

			label point
			`	"But, to the point. A scout team, aboard the Polari-class scout 'Silver Albedo,' was sent to survey the Twirl system two weeks ago, but has missed its check-in by multiple days. To reach that star cluster and return would normally require a trip of weeks, all through perilous territory, and I am ashamed to say that we could be forced to consider writing off the scouts as lost. But I have heard that the visiting human, Captain <last>, is in possession of a new kind of faster-than-light drive, one that can travel without the need for starways. Tell me, is this true, question?"`
			`	You nod.`
			`	"Excitement. Then, the trip could be shortened to mere days, giving us the possibility of mounting a rescue. I will mark the location of the Twirl system on your map, though it is probable that the Silver Albedo is elsewhere in the star cluster. The highest priority is of course the safe recovery of its crew, though if you could recover any data or samples they have managed to gather, that would also be much appreciated."`
				accept

	npc assist
		government "Avgi"
		personality derelict mute pacifist target
		system Whirlwind

> 		ship "Polari (Derelict)" "Silver Albedo"

Reviewer 5: As it stands, once you board the Silver Albedo it is repaired and can go wherever it wants. Perhaps make it self-destruct...?

		system "Corporeal"
		fleet
			names "aberrant names"
			variant
				"Aberrant Junior"
				"Aberrant Traincar"
	npc kill
		government "Aberrant"
		personality
			confusion 90
			coward secretive staying target unconstrained uninterested vindictive
		system "Empyrean"
		fleet
			names "aberrant names"
			variant
				"Aberrant Hugger"
				"Aberrant Pileup"
	npc kill
		government "Aberrant"
		personality
			confusion 90
			coward secretive staying target unconstrained uninterested vindictive
		system "Firmament"
		fleet
			names "aberrant names"
			variant
				"Aberrant Longfellow" 2
				"Aberrant Dancer"
	npc kill
		government "Aberrant"
		personality
			confusion 90
			coward secretive staying target unconstrained uninterested vindictive
		system "Lumine"
		fleet
			names "aberrant names"
			variant
				"Aberrant Latte"
				"Aberrant Chomper" 2
				"Aberrant Pileup"

	on complete
		conversation
			`You receive a hero's welcome back on the Feo Platform from the Twilight Guards stationed there. After a short celebration and many expressions of praise toward you, an alien stranger, you find Darius and Sora waiting by the <ship> to speak with you.`

> 			`	"<first>, I think we have a golden opportunity," he says. "I was just talking to an Assemblyman, Hriso, who's worried about the political situation in the eastern branch of the Tangled Shroud.`

Reviewer 5: Ah, okay. Nevermind, sorry.

	source "Outpost Tekis"
	passengers 3
	blocked "You'll need three free bunks on the <ship> to transport Darius, Sora, and Leon."
	destination "Valhalla"
	to offer
		has "Avgi: Dissonance Outreach: done"
	on enter "Decet"
		conversation
			`	Just as your jump drive reacted strangely when you first entered Avgi space, it does so again, but with an inverse effect. Your instrumentation say it is much warmer than usual, almost hot enough to be dangerous. Perhaps your jump drive's range has changed as well.`
	on enter
		system
			attributes "core" "deep" "dirt belt" "near earth" "rim" "south"
		conversation
			`	As the purple flames of your jump drive fade, Darius, Sora, and Leon are all speechless. For the first time in six years, they are back in human space.`
			`	"<first>," says Darius in a shaky voice. "Take us to Valhalla, please. I need to see my home."`
	on complete
		clear "avgi: twilight trapped"
		"reputation: Avgi (Dissonance Aggressive)" <?= -10
		conversation
			`	You land in Valhalla's main spaceport, and Darius practically jumps out of your ship before it can even touch down. "I'll be in contact, <first>, but I have to tell someone that I'm still alive."`

> 			`	Now that you know of a way both in and out of the Twilight, you can visit the Avgi without fear of being trapped again.`

Reviewer 5: Do you plan on including an alternative once the Successors are in, so that one can take the path out that leads through the Tangles via Eta Carinae? Or should that be left up to the player? Additionally, any provisions for someone with metaknowledge of either path taking a beeline out?

Reviewer 5: ...Wait, okay, this mission probably runs fine if you go via Eta Carinae.

			die
			label next
			`	The aliens hum and chirp excitedly amongst themselves, the sound reminding you of a string instrument. You notice they seem to produce sounds not from any particular vocal organs, but by rubbing their forearms or legs together akin to a cricket. One of the aliens seems to hush the others, flaring their wings out before returning them to a folded position on their back. They then raise a curved, black instrument reminiscent of a violin bow, drawing it over one of their forelimbs for a moment before the device begins to speak in a synthetic, vaguely masculine voice.`
			`	"Invitation. Welcome, visiting human. I am Kitrino of the Avgi, the one with whom you spoke earlier. I would show you to our leaders to discuss your arrival to our space."`
			`	"I'm Captain <first> <last>," you say.`
			choice
				`	"How can you speak my language?"`
				`	"Who are those hostile aliens back toward the north?"`
				`	"Tell me about your civilization."`

			`	"Reassurance. All of your questions will be answered in time, Captain <last>. Please, now follow."`

> 			`	You are led through the hangar, an entourage of guards separating you from curious onlookers - pilots, mechanics, or soldiers, you aren't sure. It's a bit awkward to be corraled by a group of people whose height barely comes past your knees, but you allow yourself to be escorted nonetheless. You exit through a ramp set in the floor, sloping downwards, then follow a long passageway that seems to perpetually curve upwards. You're reminded of older model space stations, ones that relied on spin gravity before gravity plating became commonplace.`

Reviewer 6: ```suggestion
			`	You are led through the hangar, an entourage of guards separating you from curious onlookers - pilots, mechanics, or soldiers, you aren't sure. It's a bit awkward to be corralled by a group of people whose height barely comes past your knees, but you allow yourself to be escorted nonetheless. You exit through a ramp set in the floor, sloping downwards, then follow a long passageway that seems to perpetually curve upwards. You're reminded of older model space stations, ones that relied on spin gravity before gravity plating became commonplace.`
```

			`	"Or, you may find yourself trapped here, just as they are. In any case, we will of course make you as comfortable as we can, as an honored guest, and treat you as we would any foreign dignitary."`
			`	"I have a Jump Drive that lets me travel beyond the hyperlanes, but it won't let me jump back to human space," you say.`
			`	"Regret. Then the situation is most unfortunate," she says. "Commitment. We will offer you what help we can, but I did not wish to lie to you about their predicament, or provide you with false hope."`
			`	"Reservation. Perhaps Captain <last> will be able to find a solution." says Galano. "I would not dismiss the possibility too quickly."`
			choice
				`	"While I'm here, can I buy your technology?"`
			`	Galano says, "Interest. I would expect human technology to be superior to our own in most aspects, but we will of course allow you to purchase items with your human credits, which we will back with our own currency with the agreement of the Assembly. In the event that it becomes possible to open trade relations with your human government, we may find it useful to have a small stockpile of currency available."`

> 			`	"Amusment. Not to mention the goodwill such a gesture could buy us," says Asime.`

Reviewer 6: ```suggestion
			`	"Amusement. Not to mention the goodwill such a gesture could buy us," says Asime.`
```

			`	Following the instructions of the station's control authority, you fly through the station's open frame to its small pressurized area, which houses the construction workers and maintenance crews. You've been told that word of your arrival has been sent ahead of you, so you aren't entirely surprised to see a middle-aged human waiting in the hangar, a look of pure astonishment on his face as your ship settles into a free space. You disembark the <ship> to find Darius waiting right outside the hatch.`
			choice
				`	"Hello."`
				`	"Hey there."`
				`	"I come in peace."`
			`	The man blinks in surprise, then extends a hand, which you shake. "So, another one of us here in Fairyland. I don't know if I should be glad to see another friendly face, or feel bad that another soul is stranded here along with us." He shakes his head. "Let's find somewhere to talk. If you don't mind telling me, how'd you end up here anyway?"`
			choice
				`	"I have a Jump Drive."`
			`	He somehow manages to look even more astonished as you walk. "A Jump Drive? Where the hell did you get one of those?"`
			choice
				`	"I took it from some hostile aliens near the Core."`
					goto korath
					to display

> 						"reputation: Korath" < -10

Reviewer 7: Corner Case: You are currently in the middle of bringing Rek to the exiles, so they are friendly, but you stole a Jump Drive from them earlier.

				`	"I took it from some aliens in the Deneb system."`
					goto pug
					to display

> 						has "FWC Pug 2D: done"

Reviewer 7: Corner Case: You stole a Jump Drive from the Pug as soon as they attacked Silver in FWC Pug 1, then went straight to the Avgi.

				`	"I got it from the Syndicate, who took it from an alien race near the Core."`
					goto korath
					to display
						has "FW Pug 2B: done"

> 				`	"I took it from some angry space squirrels."`

Reviewer 8: This doesn't quite work, if you just passed through the uhai, and didn't do any Wanderers story, this wouldn't make sense (as far as you know, the hai don't have jump drives). You should require the the first wanderer mission where the Hai having jump drives is mentioned

					goto unfettered
					to display
						has "visited system: Ehma Ti"
				`	"I took it from the Quarg."`
					goto quarg
					to display

> 						"reputation: Quarg" < 0

Reviewer 7: Edge Case: You reputation farmed the Quarg so they are still friendly even after you disabled one of their ships and stole a Jump Drive.

Reviewer 7: But you could still have just taken only the jump drive, though that would probably be more of a corner case than an edge case.

				`	"I'm afraid I can't tell you that."`
			`	He looks skeptical. "Alright then, keep your secrets."`
				goto next

			label korath
			`	"Aliens other than the Quarg?" You nod. "Huh. I've heard rumors about alien attacks in the Core before, but I always thought it was the Syndicate making excuses for piracy."`
				goto next

			label pug

> 			`	"Deneb? There's no hyperlanes to that system, are there? Strange." He looks thoughtful for a moment. "I've heard stories before, of... nevermind, excuse me."`

Reviewer 6: ```suggestion
			`	"Deneb? There's no hyperlanes to that system, are there? Strange." He looks thoughtful for a moment. "I've heard stories before, of... never mind, excuse me."`
```

				goto next

			label unfettered
			`	"The Hai have Jump Drives? I didn't think their technology was that advanced." He looks thoughtful. "Unless they got them from someone else, that is."`
				goto next

			label quarg
			`	"...What? You're kidding, right? How the hell did you defeat a Quarg ship?" He looks worried for a second, taking a step back. "Do I even want to be seen associating with you?" He laughs nervously.`
				goto next

			label next
			`	Darius shakes his head for a moment, then looks back at you. "Well, since you're still here, I'm assuming your Jump Drive couldn't get you out of here?" You tell him about how you were unable to jump back to human space, and he nods along.`

> 			`	"There has to be a way out, I just know it. But in six years, we've only been able to spend a few days at a time in the systems bordering human space, before those damn space shoggoths forced us back. And," he lowers his voice, looking around surreptitiously, "I don't think the Avgi are that enthusiastic about us trying to map the place either. I don't think they're exactly trying to keep us here or anything, but they've been pretty insistant on us humans not getting ourselves killed doing something stupid, like say, desperately trying to find a way back to our homes." He sounds frustrated.`

Reviewer 6: ```suggestion
			`	"There has to be a way out, I just know it. But in six years, we've only been able to spend a few days at a time in the systems bordering human space, before those damn space shoggoths forced us back. And," he lowers his voice, looking around surreptitiously, "I don't think the Avgi are that enthusiastic about us trying to map the place either. I don't think they're exactly trying to keep us here or anything, but they've been pretty insistent on us humans not getting ourselves killed doing something stupid, like say, desperately trying to find a way back to our homes." He sounds frustrated.`
```

					goto end

			label end
			`	You sit together in silence for a few more minutes, when Darius seems to receive a message on his communicator. "I get patched in to the emergency alert system," he explains as he reads it. Suddenly, he pales. "There's a massive invasion headed towards the Ensemble system. Dozens of Aberrations, flying straight into the Shroud." He looks up at you. "They don't normally try to go too deep in the Shroud. Something about it spooks them, I think, but Ensemble is shallow enough that they've attacked it before. Nothing of this scale though, and... damn, Sora's on Weledos, the moon in the Ensemble system."`
			`	He looks at you, hopeful. "We've got to do something. The Avgi are assembling a relief force around Napigeo Station, their big shipyard near here. Please, join us."`
				accept

mission "Avgi: Defend Ensemble 2"
	name "Ensemble Defense"
	description "Assist the Avgi in the Ensemble system with an Aberrant deep raid."
	source "Napigeo Station"
	landing
	destination "Weledos"
	to offer
		has "Avgi: Defend Ensemble 1: done"
	on offer
		conversation

> 			`	Napigeo Station is buzzing with activity as you pull into one of the few open berths available. Avgi ships of all sizes, painted in the blue-green of the Twilight Guard, launch every few minutes, replaced moments later by battle-scarred warships that slip into drydock for repair.`

Reviewer 6: ```suggestion
			`	Napigeo Station is buzzing with activity as you pull into one of the few open berths available. Avgi ships of all sizes, painted in the blue-green of the Twilight Guard, launch every few minutes, replaced moments later by battle-scarred warships that slip into dry dock for repair.`
```

			`	You enter a large, round conference room with walls of polished rock. Waiting there are a handful of Avgi of various ages, who immediately begin conversing with Hriso in their own language.`
			choice
				`	"What are they saying?"`
				`	(Remain silent.)`
			`	Hriso turns to you. "Consideration. I suppose you might be curious about what we are discussing. The local mining guild here says they cannot sustain their operations with the low price on iron mandated by the Consonance, as we are not buying enough rarer metals like titanium to make up for it."`
			`	"That's stupid," says Sora. "Without cheap steel we wouldn't be able to keep expanding Peripheria or build the Aktina Cylinder."`
			`	Hriso doesn't translate that, returning to conversing with the miners. Sora pulls out a translator of her own, listening intently to the conversation, but Darius pulls you aside. "I had a talk with a few people on the way here, and they say there's someone who knows more about the first days after Incursion. Come on."`
			`	"What did they say?" you ask.`

> 			`	"Apparently it's someone we have to meet in person" he grumbles. "I don't like it, but I don't think it's anything dangerous. Just inconvenient." You excuse yourself and walk along with Darius.`

Reviewer 2: ```suggestion
			`	"Apparently it's someone we have to meet in person," he grumbles. "I don't like it, but I don't think it's anything dangerous. Just inconvenient." You excuse yourself and walk along with Darius.`
```

> 			`	Before long, you find yourself in some kind of mess hall, dozens of Avgi sipping from colorful drinks. Darius freezes as he spots a short man seated on a crude stool, the only human-style seating in the room, together with a few Avgi.`

Reviewer 2: ```suggestion
			`	Before long, you find yourself in some kind of mess hall, with dozens of Avgi sipping from colorful drinks. Darius freezes as he spots a short man seated on a crude stool, the only human-style seating in the room, together with a few Avgi.`
```

			`	He pauses, taking a drink of what looks like water. "They said they had found a gateway of sorts, an ancient alien artifact built by an unknown Predecessor species. Somehow, they had activated it, and that's where the first Aberrant came from."`
			choice
				`	"And you want to use this gateway as a way out?"`
				`	"Where do you think this gateway leads?"`
			`	"I don't know where it leads, but it'll be somewhere other than Avgi space. Maybe somewhere your Jump Drive can work, which would let us eventually make our way back to human space. But it's better than sitting here and twiddling our thumbs for another six years."`
			`	"Where is this gateway?" Darius asks.`
			`	"The Aberrant came from the East, beyond the furthest reaches of Avgi space. A place the Avgi always had trouble exploring, due to strange energy storms periodically ionizing entire systems. But it's apparently recognizable by the red nebula suffusing it, just as the Twilight is surrounded by a cluster of blue nebulas."`

> 			`	"There's no point in waiting, then. If there's a chance, even a small one, we should check it out," says Darius, looking hopeful for the first time since you've met him.`

Reviewer 2: Maybe there should be some option here to tell them about the Ember Waste if you’ve found it?

	destination "Valhalla"
	to offer
		has "Avgi: Dissonance Outreach: done"
	on enter "Decet"
		conversation
			`	Just as your Jump Drive reacted strangely when you first entered Avgi space, it does so again, but with an inverse effect. Your instrumentation say it is much warmer than usual, almost hot enough to be dangerous. Maybe something has changed about your Jump Drive's range?.`
	on enter
		system
			attributes "core" "deep" "dirt belt" "near earth" "rim" "south"
		conversation
			`	As the purple flames of your Jump Drive fade, Darius, Sora, and Leon are all speechless. For the first time in six years, they are back in human space.`
			`	"<first>," says Darius in a shaky voice. "Take us to Valhalla, please. I need to see my home."`
	on complete
		clear "avgi: twilight trapped"
		"reputation: Avgi (Dissonance Aggressive)" <?= -10
		conversation
			`	You land in Valhalla's main spaceport, and Darius practically jumps out of your ship before it can even touch down. "I'll be in contact, <first>, but I have to tell someone that I'm still alive."`
			`	Now that you know of a way both in and out of the Twilight, you can visit the Avgi without fear of being trapped again.`

> 

Reviewer 2: Are all these empty lines needed? Or are they for convenience in later writing?

			`	The address appears to be out in the country, and you take in the view as the taxi smoothly glides along the roads. Unlike on Memory, the landscape here is lush and green, with rolling hills, abundant forests, and rushing streams. You find that the address belongs to a cozy-looking riverfront house, a large garden visible next to it.`
			`	Darius has never looked so nervous in his life, but you give him a gentle shove towards the door. His knocking is unanswered, and his face starts to once more slip into despair, but you hear a voice from the garden.`
			`	"Darius? Is that you?" A woman with greying hair steps onto the porch, eyes shining. "You're back... I knew you were alive!" she cries out, rushing to embrace him. The two begin speaking, faster than you can follow, as you hang back somewhat awkwardly.`
			`	After a few minutes, you try to excuse yourself, but Darius won't have it. "No, no, I owe this all to you, <first>. You're the only reason I'm here."`
			`	Katrina looks at you curiously. "Aliens, huh? You don't look like an alien."`
			choice
				`	"Huh?"`

> 				`	"That's because I'm not"`

Reviewer 9: Any reason why this was resolved? Seems like a pretty straightforward typo fix to me.

		"reputation: Avgi (Twilight Guard)" += 10
		set "discovered avgi"
		log `Met an alien species calling themselves the Avgi. Their home cluster appears infested with hostile ships from some unknown advanced civilization. Made contact with a scout ship and followed it back through a region south of Avgi space called the Tangled Shroud. Met with members of the Avgi government and made an agreement to let you purchase civilian technology.`
		log "Factions" "Avgi" `The Avgi are a young race of aliens resembling large, colorful dragonflies, which inhabit the Twilight Nebula in this part of the galaxy. After an overwhelming invasion by ships they call the Aberrant, most of their surviving spacecraft fled south through a maze-like region called the Tangled Shroud.`
		log "Factions" "Avgi Consonance" `The exiled Avgi seem to be led by a central government called the Consonance, but there are many who do not recognize this government.`
		log "People" "Asime" `Asime is one of the three First Speakers of the Consonance Assembly. While reserved, Asime appears to be defensive about the decisions made by the Consonance in response to the Aberrant incursion.`

> 		log "People" "Galano" `Asime is one of the three First Speakers of the Consonance Assembly. He appears to be an elder statesman, with much pride for what the Avgi have accomplished.`

Reviewer 2: ```suggestion
		log "People" "Galano" `Galano is one of the three First Speakers of the Consonance Assembly. He appears to be an elder statesman, with much pride for what the Avgi have accomplished.`
```

> 		log "People" "Prasinos" `Asime is one of the three First Speakers of the Consonance Assembly. She seems to take a negative view on the Avgi history of division and fractiousness.`

Reviewer 2: ```suggestion
		log "People" "Prasinos" `Prasinos is one of the three First Speakers of the Consonance Assembly. She seems to take a negative view on the Avgi history of division and fractiousness.`
```

			`	Prasinos says, "Assurance. Of course, we will introduce your fellow humans to you. But..." he trails off.`
			`	Asime continues for him. "Apprehension. The humans that came here before you, I must confess are still here for a reason. Perhaps they can give more details, but they have told us that when they first entered our space, they immediately attempted to leave. For whatever reason, they were unable to. I do not know the means by which you were able to travel to our space, but perhaps you will be able to return, and take these stranded humans with you."`
			`	"Or, you may find yourself trapped here, just as they are. In any case, we will of course make you as comfortable as we can, as an honored guest, and treat you as we would any foreign dignitary."`
			choice

> 				`	"I have a jump drive that lets me travel beyond the hyperlanes, but it won't let me jump back to human space," you say.`

Reviewer 7: ```suggestion
				`	"I have a jump drive that lets me travel beyond the hyperlanes, but it won't let me jump back to human space."`
```

				"Aberrant Triplet (Disable-able)" 3

	on complete
		log `Defended Ensemble from the Aberrant raid with the help of a fleet of Twilight Guard ships armed with nuclear weapons. The Aberrant apparently do not usually raid this deep into the Tangled Shroud, which makes the size of this battle worrying.`
		log "People" "Sora" `Sora was part of the Deep expedition that became trapped in Avgi space, serving as an engineer. She has been assisting the Twilight Guard with reinforcing their defenses in the Tangled Shroud. Despite Darius' fears, she does not appear to blame him for their predicament.`

> 		log "People" "Darius Caldwell" `Despite Darius' pessimism, at least some of his crew does not blame him for their situation.`

Reviewer 2: ```suggestion
		log "People" "Darius Caldwell" `Despite Darius' pessimism, at least some of his crew do not blame him for their situation.`
```

> 		log "People" "Kokkino" `Kokkino is a Captain in the Twilight Guard, who seems interested in the possible help you can provide the Avgi..`

Reviewer 2: ```suggestion
		log "People" "Kokkino" `Kokkino is a Captain in the Twilight Guard, who seems interested in the possible help you can provide the Avgi.`
```

			has "Avgi: Scout Rescue: done"
			has "Avgi: Frontline Combat: done"
	on offer
		log "People" "Hriso" `Hriso is an Avgi diplomat, who negotiates with dissenters unhappy with Consonance rule. While loyal to the Consonance, he seems to have some sympathy for the Dissonance movement and its views.`
		log "People" "Sora" `Sora takes a negative view on the Dissonance, and those who she sees as not contributing to the war effort against the Aberrant.`
		log "Factions" "Avgi Consonance" `Membership in the Consonance varies widely between the various Avgi polities, and a significant portion of the Avgi that escaped the Aberrant do not recognize its legitimacy.`

> 		log "Factions" "Avgi Dissonance" `The Dissonance is a catchall term for the various groups and individuals who do not accept the Consonance as a unified Avgi government. Ranging from anarchists to dissenting corporations, the Dissonance is a big-tent movement rather than being is a centralized group.`

Reviewer 2: ```suggestion
		log "Factions" "Avgi Dissonance" `The Dissonance is a catchall term for the various groups and individuals who do not accept the Consonance as a unified Avgi government. Ranging from anarchists to dissenting corporations, the Dissonance is a big-tent movement rather than being a centralized group.`
```
